src/Utilities/PaymentHelpers.res

```diff
@@ -1717,3 +1713,134 @@ let paymentIntentForPaymentSession = (
     (),
   )
 }
+
+let callAuthLink = (
+  ~publishableKey,
+  ~clientSecret,
+  ~paymentMethodType,
+  ~pmAuthConnectorsArr,
+  ~iframeId,
+) => {
+  open Promise
+  let endpoint = ApiEndpoint.getApiEndPoint()
+  let uri = `${endpoint}/payment_methods/auth/link`
+  let headers = [("Content-Type", "application/json"), ("api-key", publishableKey)]->Dict.fromArray
+
```

Comment:
add logs for API request, response and errors

Reply:
`add logs for API request, response and errors`
Will do in the 2nd PR.